src/mixins/canvas_dataurl_exporter.mixin.js

```diff
@@ -85,20 +85,21 @@ fabric.util.object.extend(fabric.StaticCanvas.prototype, /** @lends fabric.Stati
 
     var croppedCanvasEl,
         croppedCtx,
+        croppedNodeCanvasEl,
         shouldCrop = 'left' in cropping ||
                      'top' in cropping ||
                      'width' in cropping ||
                      'height' in cropping;
 
     if (shouldCrop) {
-
       croppedCanvasEl = fabric.util.createCanvasElement();
+      croppedNodeCanvasEl = fabric.isLikelyNode ? canvasEl._nodeCanvas : canvasEl;
```

Comment:
It's not a big change. Canvas was never the same between browser and node. The previous version of jsdom just returned the canvas when you did ```fabric.document.createElement('canvas')```, but in the new version of jsdom, they create a wrapper around the canvas returned, but have it accessible via ```_nodeCanvas```. (https://github.com/tmpvar/jsdom/blob/master/lib/jsdom/level2/html.js#L1472)